### PR TITLE
Model for assets and mitigation types

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

**Review by mbvertex — DISMISSED** [review]

File: synapse/models/orgs.py [hunk 1]
```diff
@@ -734,6 +744,41 @@ def getModelDefs(self):
                     ('org', ('ou:org', {}), {}),
                     ('name', ('ou:name', {}), {}),
                 )),
+
+                ('ou:asset', {}, (
```
Comment (reign-git) [inline synapse/models/orgs.py:748]: Do we want to add a :first / :start and :last / :end date props to capture when the node first became an asset / was last considered an asset?

File: synapse/models/orgs.py [hunk 2]
```diff
@@ -734,6 +744,41 @@ def getModelDefs(self):
...
+                    ('priority:availability', ('meta:priority', {}), {
+                        'doc': 'The priority of protecting the availability of the asset.'}),
+
+                    ('node', ('ndef', {}), {
+                        'doc': 'The node which represents the asset.'}),
+
+                    ('place', ('geo:place', {}), {
+                        'doc': 'The place where the asset is deployed.'}),
+
+                    ('owner', ('ps:contact', {}), {
```
Comment (reign-git) [inline synapse/models/orgs.py:783]: is owner the same as manager? If not do we want to record who manages the asset?

File: synapse/models/orgs.py [hunk 3]
```diff
@@ -734,6 +748,44 @@ def getModelDefs(self):
                     ('org', ('ou:org', {}), {}),
                     ('name', ('ou:name', {}), {}),
                 )),
+
+                ('ou:asset', {}, (
+                    ('org', ('ou:org', {}), {
```
Comment (therealsilence) [inline synapse/models/orgs.py:753]: You have :org / ou:org as "The organization that owns the asset" and also :owner / ps:contact as ''contact information of the asset owner". These seem either duplicative or else I'm not sure of the distinction / we need better docstrings?

File: synapse/models/orgs.py (hunk 2 above)
Comment (invisig0th) [reply synapse/models/orgs.py:783]: The only distinction i've seen so far is the difference between who "is responsible" ( owner ) and who "has" ( operator ) the asset. I'd prefer not to head down into distinguishing different relationships between people and assets until we encounter the need :)

File: synapse/models/orgs.py (hunk 3 above)
Comment (invisig0th) [reply synapse/models/orgs.py:753]: The difference is identifying a contact for a person "responsible" for the asset vs the org which is tracking the asset. One could assume that :owner::org should link up, but i think the direct property is worth it for clarity. Thoughts?

Comment (invisig0th) [reply synapse/models/orgs.py:753]: try to clarify the doc string :+1:

File: synapse/models/orgs.py [hunk 4]
```diff
@@ -734,6 +756,51 @@ def getModelDefs(self):
...
+
+                ('ou:asset:type:taxonomy', {}, ()),
+                ('ou:asset', {}, (
+                    ('org', ('ou:org', {}), {
+                        'doc': 'The organization which owns the asset.'}),
+
+                    ('id', ('str', {'strip': True}), {
+                        'doc': 'The ID of the asset.'}),
+
+                    ('name', ('str', {'strip': True, 'onespace': True}), {
```
Comment (Cisphyx) [inline synapse/models/orgs.py:768]: Don't need strip with onespace
```suggestion
                    ('name', ('str', {'onespace': True}), {
```

Comment (invisig0th) [reply synapse/models/orgs.py]: Thanks, i keep forgetting that :)

**Review by MichaelSquires — APPROVED** [review]

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**